libs/datamodel/core/tests/types/postgres_native_types.rs

```diff
@@ -1,6 +1,21 @@
 use crate::common::*;
+use crate::types::helper::test_native_types_with_field_attribute_support;
 use datamodel::{ast, diagnostics::DatamodelError};
 
+#[test]
+fn should_fail_on_serial_data_types_with_number_default() {
+    fn error_msg(type_name: &str) -> String {
+        format!(
+            "Sequential native type {} of Postgres must not have a static default value.",
+            type_name
+        )
+    }
+
+    for tpe in vec!["SmallSerial", "Serial", "BigSerial"] {
```

Comment:
nitpick: `&["SmallSerial", "Serial", "BigSerial"]` would also work

Reply:
Beat me to it :) Tom's suggestion doesn't allocate memory at runtime, which is nice.